app/resources/messages.py

```diff
@@ -97,18 +97,58 @@ def _message_save(self, message, is_draft, draft_id):
 
         if is_draft is True:
             Modifier().del_draft(draft_id)
-        return MessageSend._alert_recipients(message.data.msg_id, message.data.thread_id)
 
-    @staticmethod
-    def _alert_recipients(msg_id, thread_id):
-        """used to alert user once messages have been saved"""
-        recipient_email = settings.NOTIFICATION_DEV_EMAIL  # TODO change this when know more about party service
-        alert_user = AlertUser()
-        alert_status, alert_detail = alert_user.send(recipient_email, msg_id)
-        resp = jsonify({'status': '{0}'.format(alert_detail), 'msg_id': msg_id, 'thread_id': thread_id})
-        resp.status_code = alert_status
+        # listener errors are logged but still a 201 reported
+        MessageSend._alert_listeners(message.data)
+
+        resp = jsonify({'status': '201', 'msg_id': message.data.msg_id, 'thread_id': message.data.thread_id})
+        resp.status_code = 201
         return resp
 
+    @staticmethod
+    def _alert_listeners(message):
+        """used to alert user and case service once messages have been saved"""
+        try:
+            MessageSend._try_send_alert_email(message)
+            MessageSend._inform_case_service(message)
+        except Exception as e:
+            logger.error('Uncaught exception in Message.alert_listeners: {}'.format(e))
+
+    @staticmethod
+    def _try_send_alert_email(message):
+        """Send an email to recipient if appropriate"""
+        party_data = None
+        if message.msg_to[0] != constants.BRES_USER:
```

Comment:
For cases where the to or from is the internal user then value 'BRES' is assumed . If this is not the case then we have much wider issues ? Can you point o cases where the internal user is not 'BRES' ?